sktime/transformations/panel/rocket/_minirocket_multivariate_variable.py

```diff
@@ -273,6 +273,45 @@ def _transform(self, X, y=None):
         set_num_threads(prev_threads)
         return pd.DataFrame(X_)
 
+    @classmethod
+    def get_test_params(cls, parameter_set="default"):
+        """Return testing parameter sets for the estimator.
+
+        Parameters
+        ----------
+        parameter_set : str, default="default"
+            Name of the set of test parameters to return, for use in tests. If no
+            special parameters are defined for a value, will return `"default"` set.
+
+        Returns
+        -------
+        params : dict or list of dict, default={}
+            Parameters to create testing instances of the class.
+            Each dict are parameters to construct an "interesting" test instance, i.e.,
+            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`.
+        """
+        params = [
+            {
+                "num_kernels": 84,
+                "random_state": None,
+                "max_dilations_per_kernel": 32,
+                "reference_length": max,
+                "pad_value_short_series": None,
+                " n_jobs": 1,
+            },
+            {
+                "num_kernels": 84,
+                "random_state": None,
+                "max_dilations_per_kernel": 32,
+                "reference_length": max,
+                "pad_value_short_series": None,
+                " n_jobs": 1,
```

Comment:
this is the reason why the CI is failing.
you have a space in front of ` n_jobs`

<img width="888" height="650" alt="image" src="https://github.com/user-attachments/assets/36b7e6e4-95c6-496a-8b80-fd86ca4246d2" />